utilities/ipplan-import.py

```diff
@@ -0,0 +1,181 @@
+"""
+NIPAP import script for IPplan data.
+
+This script uses the standard export files base.txt and ipaddr.txt from Ipplan.
+"""
+import configparser
+import os
+# use local pynipap, useful if we are developing
+import sys
+sys.path.append('../pynipap')
+import pynipap
+import csv
+import ipaddress
+
+
+def add_ip_to_net(networks, host):
+    """
+    Add hosts from IPPlan to networks object.
+    """
+    for network in networks:
+        if host['ipaddr'] in network['network']:
+            network['hosts'].append(host)
+            return
+
+
+def get_networks(base_file, ipaddr_file):
+    """
+    Gather network and host information from IPPlan export files.
+    """
+    networks = []
+
+    base = open(base_file, 'r')
+
+    csv_reader = csv.reader(base.readlines(), delimiter='\t')
+    for row in csv_reader:
+
+        # Rows with less than 3 entries is most likely a newline.
+        if len(row) < 3:
+            continue
+
+        network = {
+            'network': ipaddress.ip_network("{}/{}".format(row[0], row[2])),
+            'comment': row[1],
+            'hosts': []
+        }
+
+        networks.append(network)
+
+    base.close()
+
+    ipaddr = open(ipaddr_file, 'r')
+
+    csv_reader = csv.reader(ipaddr.readlines(), delimiter='\t')
```

Comment:
I don't know csv_reader by heart but do you really need to do readlines() here? Isn't enough with just feeding it the file handle and it will call __next__() on it? That way you don't load the entire file into memory.